### PR TITLE
Add Trino summit recap blog post about Starburst Galaxy

### PR DESCRIPTION
**Pull request by monimiller**
Add Trino summit recap blog post about Starburst Galaxy

### CODE REVIEW

**Comment by mosabua** [comment]
Need to squash all commits into one and have a message like

Add Trino summit recap blog post about Starburst Galaxy

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+---
+
+I have been reading all the wonderful posts by our awesome
+speakers, facilitated by Brian and Cole, in the
+[Trino Summit 2022 recap post series]({% post_url
+2022-11-21-trino-summit-2022-recap %}). Because I have a perpetual fear of
+missing out, I convinced the two of them that I should get in on the fun.
+For this latest installment in the series, I will be recapping my very own Trino
+Summit talk. Basically, I'm ripping off Bo Burnham's comedy bit
+where he [reacts to his own reaction video](https://youtu.be/FZVMB8mrNO0?t=35),
```
Comment (bitsondatadev) [inline _posts/2022-12-12-trino-summit-2022-starburst-recap.md:16]: 🤣

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+table that is cleaned and ready to be utilized by a data consumer. Since I only
+want account administrators to have access to update the consume layer, I
+demonstrate applying role-based access control capabilities within Starburst
+Galaxy, which provides the proper data governance to satisfy this request. I
+then create some visualizations to analyze which Pokemon are common to spawn in
+the San Francisco Bay Area.
+
+I walk through all the setup required to put this data lakehouse architecture
+into action including creating my catalogs, clusters, schemas, and tables. 
+After incorporating open-table formats, applying native security, and building
```
Comment (bitsondatadev) [inline _posts/2022-12-12-trino-summit-2022-starburst-recap.md:56]:
```suggestion
After incorporating open table formats, applying native security, and building
```

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+S3, which contains encounter information of each Pokemon. This includes the
+geo-location data, aka the latitude and longitude, of where each Pokemon spawned
+and how long the Pokemon was at that location. What we don’t have is any
+information on that Pokemon’s abilities, that’s all contained in the Pokédex
+stored in MongoDB which I’ve cleverly nicknamed **PokeMongoDB**. This has all
+the statistics on your desired Pokemon including type, legendary status,
+catch rate, and more. To make meaningful information out of our data, we need to
+combine the incoming geo-location data with the static dimension CSV table
+located in MongoDB.
+
```
Comment (bitsondatadev) [inline _posts/2022-12-12-trino-summit-2022-starburst-recap.md:42]: I would add the architecture diagram here.

**Review by bitsondatadev — COMMENTED** [review]
Few small changes then it looks good!

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+into action including creating my catalogs, clusters, schemas, and tables. 
+After incorporating open-table formats, applying native security, and building
+out a reporting structure, I have confidence that my data lakehouse is built
+to last (and end up with some really cool final Pokemon graphs). I'm working on
+turning this demo into a tutorial, and will update the post when this happens.
+In the mean time, if you are curious about trying
+Starburst Galaxy, I invite you to look at one of the [already created tutorials](https://docs.starburst.io/starburst-galaxy/tutorials/index.html).
+
+
+## Datanova 2023
```
Comment (mosabua) [inline _posts/2022-12-12-trino-summit-2022-starburst-recap.md:64]: I think this section is not appropriate for the individual recap post. We could add something like this in the final post with the teaser/fomo video as an outlook on whats next when we talk about next Trino summit. But here.. I would remove. Ultimately however I think @martint should make the call in this instance.

**Comment by mosabua** [comment]
Have not done specific review but at first glance you should definitely link to the Starburst Galaxy sign up page, the product page and the documentation.

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md (hunk 4 above)
Comment (bitsondatadev) [reply _posts/2022-12-12-trino-summit-2022-starburst-recap.md:64]: Agreed, let's defer to Martin.

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Federating them all on Starburst Galaxy"
+author: "Monica Miller"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/starburst.jpg
+---
+
+I have been reading all the wonderful posts by our awesome
```
Comment (colebow) [inline _posts/2022-12-12-trino-summit-2022-starburst-recap.md:9]: I don't know how strongly I feel about it, but my thought is that it _might_ be good to introduce yourself in this first paragraph if the post is first-person perspective.

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+out a reporting structure, I have confidence that my data lakehouse is built
+to last (and end up with some really cool final Pokemon graphs).
+
+## Helpful links
+
+- Sign up for [Starburst Galaxy](https://www.starburst.io/platform/starburst-galaxy/start/)
+- Read the [docs](https://docs.starburst.io/starburst-galaxy/index.html)
+- Try a
+  [tutorial](https://docs.starburst.io/starburst-galaxy/tutorials/index.html) for yourself
+- Register for [Datanova](https://www.starburst.io/datanova/?utm_source=event&utm_medium=datanova&utm_campaign=[…]Event-Datanova-social-promo&utm_content=trinosummitrecapblog)!
```
Comment (bitsondatadev) [inline _posts/2022-12-12-trino-summit-2022-starburst-recap.md:68]: Small nit for consistency

```suggestion
- Register for [Datanova](https://www.starburst.io/datanova/?utm_source=event&utm_medium=datanova&utm_campaign=[…]Event-Datanova-social-promo&utm_content=trinosummitrecapblog)
```

**Review by bitsondatadev — APPROVED** [review]
One little change then LGTM!

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+<!--more-->
+
+{% youtube Zfmxwu0m98k %}
+
+
+## Recap
+
+The premise of this example is that we have Pokemon Go data being ingested into
+S3, which contains encounter information of each Pokemon. This includes the
```
Comment (colebow) [inline _posts/2022-12-12-trino-summit-2022-starburst-recap.md:33]:
```suggestion
S3, which contains encounter information for each Pokemon. This includes the
```
Also, it's technically Pokémon if you wanna add the accent on the e... but that's fine to exclude. Most people skip it.

**Review by colebow — APPROVED** [review]

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+stored in MongoDB which I’ve cleverly nicknamed **PokeMongoDB**. This has all
+the statistics on your desired Pokemon including type, legendary status,
+catch rate, and more. To make meaningful information out of our data, we need to
+combine the incoming geo-location data with the static dimension CSV table
+located in MongoDB.
+
+<img src="/assets/blog/trino-summit-2022/starburst-architecture.png"/>
+
+To do this, I build out a reporting structure in the data lake using
+Starburst Galaxy. The first step is to read the data in the land layer, then
```
Comment (mosabua) [inline _posts/2022-12-12-trino-summit-2022-starburst-recap.md:46]: whats a "land layer"

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+clean and optimize that data into more performant orc files in the structure
```
Comment (mosabua) [inline _posts/2022-12-12-trino-summit-2022-starburst-recap.md:47]: ORC files

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+layer. Finally, I join the spawn data & Pokédex data together into a single
```
Comment (mosabua) [inline _posts/2022-12-12-trino-summit-2022-starburst-recap.md:48]: spawn data and Pokedex data

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+table that is cleaned and ready to be utilized by a data consumer. Since I only
+want account administrators to have access to update the consume layer, I
```
Comment (mosabua) [inline _posts/2022-12-12-trino-summit-2022-starburst-recap.md:50]: Break up this long sentence

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+demonstrate applying role-based access control capabilities within Starburst
+Galaxy, which provides the proper data governance to satisfy this request. I
+then create some visualizations to analyze which Pokemon are common to spawn in
+the San Francisco Bay Area.
```
Comment (mosabua) [inline _posts/2022-12-12-trino-summit-2022-starburst-recap.md:54]: area

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2022-12-12-trino-summit-2022-starburst-recap.md (hunk 5 above)
Comment (monimiller) [reply _posts/2022-12-12-trino-summit-2022-starburst-recap.md:9]: I was trying to make the first paragraph my subtle intro without being like "hi im monica"

**Comment by bitsondatadev** [comment]
Looks good! Great work!

**Pull request merged**